Packs/Zscaler/Integrations/Zscaler/Zscaler.py

```diff
@@ -1016,13 +1017,220 @@ def set_user_command(args):
         return responseJson
 
 
+def create_ip_destination_group(args: dict):
+    payload = dict()
+    payload['name'] = args.get('name', '')
+    payload['type'] = args.get('type', '')
+    payload['countries'] = argToList(args.get('countries', ''))
+    payload['ipCategories'] = argToList(args.get('ip_categories', ''))
+    payload['description'] = args.get('description', '')
+    payload['addresses'] = argToList(args.get('addresses', ''))
+    payload['isNonEditable'] = args.get('is_non_editable', False)
```

Comment:
This can be refactored into 
```
payload = {
    'name': args.get('name', ''),
    'type': args.get('type', ''),
    'countries': argToList(args.get('countries', '')),
    'ipCategories': argToList(args.get('ip_categories', '')),
    'description': args.get('description', ''),
    'addresses': argToList(args.get('addresses', '')),
    'isNonEditable': args.get('is_non_editable', False)
}
```

This converts the dictionary creation and assignment into one execution, which is more concise.

Reply:
done. :)